apple/macbook-pro/11-5/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, config, pkgs, ... }:
+
+let
+
+  kernelPackages = config.boot.kernelPackages;
+
+in
+
+{
+  imports = [
+    ../.
+    ../../../common/pc/laptop/ssd
+    <nixpkgs/nixos/modules/hardware/network/broadcom-43xx.nix>
+  ];
+
+  boot.loader.systemd-boot.enable = true;
+  boot.loader.efi.canTouchEfiVariables = true;
```

Comment:
This should be only done for the first installation. Otherwise a new boot entry is created for every nixos-rebuild switch. This can fill up the nvram, which is also limited in terms of write-cycles.

Reply:
Interesting, thank you. How about if I put these two lines as instructions in the README.md file instead?

I suppose [macbook-pro/10-1](https://github.com/NixOS/nixos-hardware/blob/master/apple/macbook-pro/10-1/default.nix#L11) could also do with a PR to fix.

Would be nice to update the [docs](https://github.com/NixOS/nixpkgs/blob/80b5f5227e4a801bc9da70ba68d59aad54618e7c/nixos/modules/system/boot/loader/efi.nix#L11) too.

Reply:
Are these instructions better suited? https://nixos.wiki/wiki/Bootloader#Keeping_kernels.2Finitrd_on_the_main_partition

Reply:
Ok, sorry for the noise: Seeing as this config is already automatically [generated by nixos-generate-config](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/installer/tools/nixos-generate-config.pl#L534) I suppose it really shouldn't be in this module at all. 

I'll remove the boot loader completely if there's no objections.